api/tests/test_asset_api.py

```diff
@@ -188,6 +188,47 @@ def test_admin_can_filter_asset_by_asset_status(self, mock_verify_id_token):
         )
         self.assertEqual(response.data["count"], count + 1)
 
+    @patch("api.authentication.auth.verify_id_token")
+    def test_admin_can_filter_asset_by_asset_department(self, mock_verify_id_token):
```

Comment:
Perhaps a more descriptive name would be `test_admin_can_filter_asset_by_assignee_department`